src/PostalRegistry.Api.Oslo/PostalInformation/Responses/PostalInformationListOsloResponse.cs

```diff
@@ -18,44 +18,56 @@ public class PostalInformationListOsloResponse
         [DataMember(Name = "@context", Order = 0)]
         [JsonProperty(Required = Required.DisallowNull)]
         [JsonConverter(typeof(PlainStringJsonConverter))]
-        public object Context => @"{ 
-            ""identificator"": ""@nest"",
-            ""id"": ""@id"",
-            ""versieId"": {
-                ""@id"": ""https://data.vlaanderen.be/ns/generiek#versieIdentificator"",
-                ""@type"": ""http://www.w3.org/2001/XMLSchema#string""
-            },   
-            ""postnamen"": {
-                ""@id"": ""https://data.vlaanderen.be/ns/adres#postnaam"",
-                ""@type"": ""@id"",
-                ""@context"": {
-                    ""straatnaam"": ""@nest"",
-                    ""geografischeNaam"": {
-                        ""@id"": ""http://www.w3.org/2000/01/rdf-schema#label"",
-                        ""@context"": {
-                            ""spelling"": ""@value"",
-                            ""taal"": ""@language""
-                        }
-                    }
-                }
-            },
-            ""postInfoStatus"": {
-                ""@id"": ""https://data.vlaanderen.be/ns/adres#Postinfo.status"",
-                ""@type"": ""@id"",
-                ""@context"": {
-                    ""@base"": ""https://data.vlaanderen.be/id/concept/postinfostatus/""
-                }
-            },
-            ""detail"": ""http://www.iana.org/assignments/relation/self"",
-            ""postInfoObjecten"": ""@graph""
-        }";
+        public object Context => @"{
+    ""@base"": ""https://basisregisters.vlaanderen.be/ns/adres"",
+    ""@vocab"": ""#"",
+    ""identificator"": ""@nest"",
+    ""id"": ""@id"",
+    ""versieId"": {
+      ""@id"": ""https://data.vlaanderen.be/ns/generiek#versieIdentificator"",
+      ""@type"": ""http://www.w3.org/2001/XMLSchema#string""
+    },
+    ""naamruimte"": {
+      ""@id"": ""https://data.vlaanderen.be/ns/generiek#naamruimte"",
+      ""@type"": ""http://www.w3.org/2001/XMLSchema#string""
+    },
+    ""objectId"": {
+      ""@id"": ""https://data.vlaanderen.be/ns/generiek#lokaleIdentificator"",
+      ""@type"": ""http://www.w3.org/2001/XMLSchema#string""
+    },
+    ""PostInfo"": ""https://data.vlaanderen.be/ns/adres#Postinfo"",
+    ""postnamen"": {
+      ""@id"": ""https://data.vlaanderen.be/ns/adres#postnaam"",
+      ""@type"": ""@id"",
+      ""@context"": {
+        ""straatnaam"": ""@nest"",
```

Comment:
This is weird...

Reply:
hhhwhy?

Reply:
What does `straatnaam` have to do with postinfo ...